adagrams/game.py

```diff
@@ -1,11 +1,129 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_LIST = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T" ],
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: ["K"],
+    8: ["J", "X"],
+    10: ["Q", "Z"]
+    }
+
+
+"""
+Create_list_of_letters function returns a list of all letters in. 
+Based on letter_pool there are total 98 letters in the list. 
+"""
+def create_list_of_letters(LETTER_POOL):
+    list_of_letters = []
+    for letter, count in LETTER_POOL.items():
+        while count > 0:
+            list_of_letters.append(letter)
+            count = count - 1
+    return (list_of_letters)
```

Comment:
I like the helper function!  One tip, docstrings go _inside_ the function:

```suggestion
def create_list_of_letters(LETTER_POOL):
    """
    Create_list_of_letters function returns a list of all letters in. 
    Based on letter_pool there are total 98 letters in the list. 
    """
    list_of_letters = []
    for letter, count in LETTER_POOL.items():
        while count > 0:
            list_of_letters.append(letter)
            count = count - 1
    return (list_of_letters)
```

Docstrings are visible to the user of your program when they run the `help` function.  :smiley:

Reply:
It is good to know, I did not know about it 😁